spark2.4/src/main/scala/KafkaReader.scala

```diff
@@ -102,11 +113,17 @@ object KafkaReader {
       reader.option("kafka.client.id", s"$group-$clientId")  // probably should use uuid instead of user input TODO
     }
     
-    val values = reader
-      .load.select(from_json(col("value") cast "string", schema, Map( "timestampFormat" -> "yyyy/MM/dd HH:mm:ss" )) as "data", col("timestamp") cast "long" as "TM_EXT_KAFKA")
-      .select("data.*", "TM_EXT_KAFKA")
-      .withColumn("TM_EXT_KAFKA", col("TM_EXT_KAFKA") * 1000)
-      .withColumn("TM_SSDS", unix_timestamp * 1000 )
+    val values = if( useFlowMarkers ) {
+      reader
+        .load.select(from_json(col("value") cast "string", schema, Map( "timestampFormat" -> "yyyy/MM/dd HH:mm:ss" )) as "data", col("timestamp") cast "long" as "TM_EXT_KAFKA")
+        .select("data.*", "TM_EXT_KAFKA")
+        .withColumn("TM_EXT_KAFKA", col("TM_EXT_KAFKA") * 1000)
```

Comment:
Why not to "merge" it with the above `select`? It's confusing.

Reply:
Updated in commit https://github.com/splicemachine/splice-machine-spark-connector/commit/43f72a64c5a3c29c4609801af6e8f428e4fde6aa